packages/config/src/config.js

```diff
@@ -111,28 +111,92 @@ function resetConfig(oldConfig) {
   return clearConfig().then(config(oldConfig))
 }
 
+/**
+ * @typedef {Object} FlowJSONv2 - Flow.json schema 'v2'
```

Comment:
This is great. Can we move this into a types folder somehow at the root level to set us up for sharing JSDoc types across the entire monorepo?

Reply:
I'll try to get this in today/tomorrow and we can consolidate it with what you are working on.